portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentExportImport.testcase

```diff
@@ -107,6 +107,97 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-114230. Verify that can import image global structures between instances."
```

Comment:
The test point is that assert the imported web content should show the image. This is different from the current description and case name. Please think over and modify the description and case name.